sql/ddl_api.sql

```diff
@@ -138,7 +138,7 @@ CREATE OR REPLACE FUNCTION  add_dimension(
     chunk_time_interval     ANYELEMENT = NULL::BIGINT,
     partitioning_func       REGPROC = NULL,
     if_not_exists           BOOLEAN = FALSE
-) RETURNS VOID
+) RETURNS TABLE(dimension_id INT, schema_name NAME, table_name NAME, column_name NAME)
```

Comment:
why isn't the middle argument the REGCLASS of the main table instead of schema_name and table name split out?

Reply:
i guess it depends who we see as users of the API which way is preferred, but the only thing returned that is really interesting is the dimension_id all other values are known to the caller of the function and can be derived from the arguments.